### DISCUSSION TITLE
스마트스토어 센터 통합 관련

### DISCUSSION THREAD

**Discussion by ikseon1026** [commerce-api-naver/commerce-solutions#63] [묻고 답하기]
안녕하세요. 몇몇 솔루션은 스마트스토어 내부에 통합 되어 메뉴가 추가되고, 세부 페이지가 존재하는 방식으로 솔루션이 개발이 이뤄진듯 합니다.
하지만, 커머스 API  및 솔루션 개발 가이드에는 스마트스토어와 통합할 수 있는 방법에 대해서는 설명이 없어서 여쭙습니다.

현재 외부 솔루션으로 개발되고 있지만, 혹시 스마트스토어 내부로 통합하여 개발할 수 내용이 설명된 자료가 있을까요? 
솔루션이 스마트스토어와 통합되어 개발된다면 사용자가 보다 편리할 듯 하여 여쭙습니다. 감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

네이버가 스마트스토어센터 기능 중 부가적인 기능을 커머스솔루션 형태로 제작하여 필요한 판매자 분들께 커머스솔루션마켓으로 서비스를 제공할 수 있습니다.
- 예시: [네이버도착보장 프로그램](https://solution.smartstore.naver.com/ko/solution/5kjYBRHOwKJiC8ExEdKnCp/detail)

그러나 네이버 이외 개발사의 커머스솔루션이 솔루션 구독으로 대상 구독자(판매자)의 스마트스토어센터에 전용 메뉴를 추가할 수 있는 기능은 현재 제공하지 않습니다.
현재의 커머스솔루션은 스마트스토어센터와 분리된 개발사 전용 시스템에서의 사용만을 지원합니다.
